src/parser/jobs/mnk/modules/Balls.tsx

```diff
@@ -0,0 +1,128 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import Color from 'color'
+import {ActionLink} from 'components/ui/DbLink'
+import {Event, Events} from 'event'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Actors} from 'parser/core/modules/Actors'
+import {Gauge} from 'parser/core/modules/Gauge'
+import {SetGauge} from 'parser/core/modules/Gauge/SetGauge'
+import {GAUGE_FADE} from 'parser/core/modules/ResourceGraphs/ResourceGraphs'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+import {fillActions} from 'utilities/fillArrays'
+import {ST_FORM_ACTIONS} from './constants'
+
+const OPO_GAUGE_COLOR = Color('#a256dc')
+const RAPTOR_GAUGE_COLOR = Color('#57b39a')
+const COEURL_GAUGE_COLOR = Color('#d7548e')
+
+const OVERCAP_SEVERITY = {
+	1: SEVERITY.MINOR,
+	3: SEVERITY.MEDIUM,
+	5: SEVERITY.MAJOR,
+}
+
+export class Balls extends Gauge {
+	static override handle = 'balls'
+	static override title = t('mnk.balls.title')`Fury Gauge`
+
+	@dependency private suggestions!: Suggestions
+	@dependency private actors!: Actors
+
+	private furyGauge = this.add(new SetGauge({
+		options: [
+			{
+				value: 'opo',
+				color: OPO_GAUGE_COLOR.fade(GAUGE_FADE),
+				label: <Trans id="mnk.balls.opoFury.label">Opo-opo Fury</Trans>,
+			},
+			{
+				value: 'raptor',
+				color: RAPTOR_GAUGE_COLOR.fade(GAUGE_FADE),
+				label: <Trans id="mnk.balls.raptorFury.label">Raptor Fury</Trans>,
+			},
+			{
+				value: 'coeurl' + 0,
+				color: COEURL_GAUGE_COLOR.fade(GAUGE_FADE),
+				label: <Trans id="mnk.balls.coeurlFury.label">Coeurl Fury 1</Trans>,
+			},
+			{
+				value: 'coeurl' + 1,
+				color: COEURL_GAUGE_COLOR.fade(GAUGE_FADE),
+				label: <Trans id="mnk.balls.coeurlFury2.label">Coeurl Fury 2</Trans>,
+			},
```

Comment:
What's the benefit of making these two separate Set Gauges rather than using a single Counter Gauge with a max value of 2?

Reply:
Currently (to my knowledge) there's no way to combine a counter gauge with a set gauge. I tried a few combinations (3 counter gauges, 2 enum gauges + 1 counter gauge, 1 set gauges + 1 counter gauge) and this one looks the best. I think the ideal solution would be to allow multiple values for each item in the set, but it's kind of a one off for Monk and nobody else.